Assets/MixedRealityToolkit.Extensions/Providers/Experimental/UnityAR/UnityARCameraSettings.cs

```diff
@@ -106,21 +108,16 @@ private void FindARFoundationComponents()
             arSessionOriginObject = GameObject.Find("AR Session Origin");
             preExistingArSessionOriginObject = (arSessionOriginObject != null);
         }
-#endif //!(WINDOWS_UWP && !ENABLE_IL2CPP)
 
         /// <inheritdoc />
-        public override async void Initialize()
+        public override void Initialize()
         {
             base.Initialize();
 
-#if !(WINDOWS_UWP && !ENABLE_IL2CPP)
-            ARSessionState sessionState = (ARSessionState)(await ARSession.CheckAvailability());
-            if (ARSessionState.Ready > sessionState)
+            if (Application.platform == RuntimePlatform.Android)
             {
-                Debug.LogError("Unable to initialize the Unity AR Camera Settings provider. Device support for AR Foundation was not detected.");
-                isInitialized = true;
+                isSupportedConfiguration = string.IsNullOrWhiteSpace(XRSettings.loadedDeviceName);
```

Comment:
Would be good to add a comment here to explain what it's doing

Reply:
I think that it would still be helpful to have a comment about the significant of the empty loadedDeviceName.

Reply:
i thought i added a comment.... will confirm

Reply:
clarified comment a bit